.github/workflows/kind-e2e.yaml

```diff
@@ -105,22 +104,6 @@ jobs:
         - e2e
 
         include:
-          # Map between K8s and KinD versions.
-          # This is attempting to make it a bit clearer what's being tested.
-          # See: https://github.com/kubernetes-sigs/kind/releases
-          #      https://hub.docker.com/r/kindest/node/tags
-        - k8s-version: v1.25.9
-          kind-version: v0.19.0
-          kind-image-sha: sha256:c08d6c52820aa42e533b70bce0c2901183326d86dcdcbedecc9343681db45161
-
-        - k8s-version: v1.26.4
-          kind-version: v0.19.0
-          kind-image-sha: sha256:f4c0d87be03d6bea69f5e5dc0adb678bb498a190ee5c38422bf751541cebe92e
-
-        - k8s-version: v1.27.1
-          kind-version: v0.19.0
-          kind-image-sha: sha256:b7d12ed662b873bd8510879c1846e87c7e676a79fefc93e17b2a52989d3ff42b
-
         # Disabled due to consistent failures
         # - ingress: gateway_istio
         #   ingress-class: gateway-api
```

Comment:
line below this one is an input that has been commented out aka disabled `test-flags`